lib/1.4/dml-builtins.dml

```diff
@@ -365,7 +365,12 @@ device is created, *before* any attributes have been
 initialized. Typically used to initialize a default value, or to set up data
 structures.
 
-The method `init` is automatically called on all objects that implement the `init` template.
+The method `init` is automatically called on all objects that implement the
+`init` template. The order in which `init` of objects are called is not
+defined, except that `init` of a particular object is guaranteed to be called
+before `init` of any of its parent objects. In particular, `init` of the device
```

Comment:
consider s/device/`device`/ (but both are ok)

Reply:
I protest against the latter, as this is not referring to the `device` object kind, template, or template type in this context, but rather, referring to the single canonical DML object representing the device.